packages/database/src/utilities/migration.js

```diff
@@ -18,14 +18,18 @@ export const insertObject = async (db, table, data, onError) =>
   db.insert(table, Object.keys(data), Object.values(data), onError);
 
 export const deleteObject = async (db, table, condition) => {
-  const [key, value] = Object.entries(condition)[0];
+  const where = Object.entries(condition)[0]
+    .map(([key, value]) => `${key} = '${value}'`)
+    .join(' AND ');
   return db.runSql(`
       DELETE FROM "${table}"
-      WHERE ${key} = '${value}'
+      WHERE ${where}
```

Comment:
Also thought it was unexpected that you can only pass one condition in to `deleteObject`, so updated that as well. Checked, and all past migrations just use one, so this doesn't have any retrospective effect on historical migrations